.github/workflows/ci.yml

```diff
@@ -1,12 +1,12 @@
-name: Ruby
+name: CI
 
-on: [push, pull_request]
+on: [push]
```

Comment:
Yeah I think we want to keep that, as @adrianna-chang-shopify mentioned it's not running the same code.

Reply:
Yeah, interesting. I didn't realize it used a separate branch. I wish the GitHub Actions showed examples. Maybe it's just me, but I feel like it could have been clearer.

Anyway, I'll undo the change. I wonder if we should ensure other projects also do the same (namely Job Iteration).